web/src/flow/providers/IFrameLogoutStage.ts

```diff
@@ -28,6 +28,8 @@ interface LogoutStatus {
 interface LogoutURLData {
     url: string;
     saml_request?: string;
+    saml_response?: string;
+    relay_state?: string;
     provider_name?: string;
     binding?: string;
 }
```

Comment:
This shouldn't need to be defined manually, they should come from the API schema

```python


class LogoutURL(PassiveSerializer):

    url = CharField()
    [ ... ]


class IframeLogoutChallenge(Challenge):
    """Challenge for iframe logout"""

    component = CharField(default="ak-provider-iframe-logout")
    logout_urls = ListField(child=LogoutURL(), default=list)


```